build.gradle

```diff
@@ -595,10 +595,10 @@ configure(rootProject) {
 
 	gitPublishCommit.dependsOn(prepareDocsForUploadToGhPages)
 	gitPublishCommit.dependsOn(createCurrentDocsFolder)
+}
 
-	task wrapper(type: Wrapper) {
-		description = 'Generates gradlew and gradlew.bat scripts'
-		gradleVersion = '4.0.2'
-	}
-
+task wrapper(type: Wrapper) {
+	description = 'Generates gradlew and gradlew.bat scripts'
+	gradleVersion = '4.1'
+	distributionType = Wrapper.DistributionType.ALL
```

Comment:
Sure, I can change it back. I personally work in gradle builds a lot at work and always finding the docs really helpful.

I'll explicitly set the distribution type to `BIN` with a mention of this comment chain.